test/fixture-scripts/index.ts

```diff
@@ -92,7 +80,7 @@ async function testFixture(fixture: FixtureInfo) {
       extensionDevelopmentPath: root,
       extensionTestsPath: join(__dirname, 'runner.js'),
       vscodeExecutablePath,
-      launchArgs: [path],
+      launchArgs: [path, '--disable-extensions'],
```

Comment:
Now that we don't need extensions, we should once again disable them all, so we don't accidentally end up relying on some behaviour (e.g., a post-save formatter) while running this locally.